### DISCUSSION TITLE
Lost account access

### DISCUSSION THREAD

**Discussion by ghost** [community/community#27500] [Other Feature Feedback, Questions, & Ideas]
I formatted my partition few days back. But, before that I didn't backup my TOTP secret key, backup code, ssh or gpg keys. Now I can't login due to 2FA. I tried to reset. But, it says can't verify device and only have ssh key verify option. I have only access to my email. This is my secondary GitHub account. Any idea on getting back my account?

**Comment by airtower-luna** [comment]
Honestly it seems unlikely, the [documentation warns](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#using-a-two-factor-authentication-recovery-code): :warning: 

> For security reasons, GitHub Support may not be able to restore access to accounts with two-factor authentication enabled if you lose your two-factor authentication credentials or lose access to your account recovery methods.

If anyone can help it's still [GitHub Support](https://support.github.com/request).

**Reply by ghost** [reply]
I tried opening ticket. And they closed the ticket saying I have to email them. But, can't send email since GitHub currently not receiving email. Also can't contact GitHub with that account since can't login.

**Reply by queenofcorgis1** [reply]
+1 to @airtower-luna - if anyone can help, it would be [GitHub Support](https://support.github.com/request), but no guarantees.

**Comment by gbdlin** [comment]
Unfortunately, it won't be possible to recover your account...

It would work against the 2 factor authentication to unlock your account without any other, equally strong authentication being performed. For some services that require your legal name, you may be able to recover your account by confirming your legal identity in some way, but GitHub doesn't collect such data.

It is the user responsibility to secure any alternative method of authentication, like backing up recovery codes, setting up another, independent 2 factor method like sms codes (even on the same phone you've set up TOTP codes on, as your phone number can be very often recovered using your legal identity, as I've mentioned above), hardware security keys, having a constant backup or another copy of your usual methods on another device, even kept in a separate location. You should never rely on a single device for 2 factor, as you'll never know when you may lose the access to it.
